### PR TITLE
Update for signed policies on HGS

### PR DESCRIPTION
**Pull request by carlosmayol**
Using same content from here: https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/security/guarded-fabric-shielded-vm/guarded-fabric-add-host-information-for-tpm-trusted-attestation.md
That I cannot find in the public docs

### CODE REVIEW

**Comment by PRMerger19** [comment]
@carlosmayol : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by rpsqrd — APPROVED** [review]

File: WindowsServerDocs/security/guarded-fabric-shielded-vm/guarded-fabric-tpm-trusted-attestation-capturing-hardware.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ For more information about the available CI policy rule levels, see [Deploy code
     ```powershell
     Add-HgsAttestationCIPolicy -Path <Path> -Name '<PolicyName>'
     ```
-
+    [!NOTE] If you're using a signed code integrity policy, register an unsigned copy of the same policy with HGS. The signature on code integrity policies is used to control updates to the policy, but is not measured into the host TPM and therefore cannot be attested to by HGS.
```
Comment (illfated) [inline WindowsServerDocs/security/guarded-fabric-shielded-vm/guarded-fabric-tpm-trusted-attestation-capturing-hardware.md:138]: Please note that the following layout is the MarkDown Note blob format used for the colored Note blobs in the docs.microsoft.com documentation pages:
```suggestion
    > [!NOTE]
    > If you're using a signed code integrity policy, register an unsigned copy of the same policy with HGS. The signature on code integrity policies is used to control updates to the policy, but is not measured into the host TPM and therefore cannot be attested to by HGS.
```

Comment (illfated) [reply WindowsServerDocs/security/guarded-fabric-shielded-vm/guarded-fabric-tpm-trusted-attestation-capturing-hardware.md:138]: … or have I missed something important regarding MD formatting in this context?

**Comment by PRMerger14** [comment]
@carlosmayol : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by rpsqrd — APPROVED** [review]

File: WindowsServerDocs/security/guarded-fabric-shielded-vm/guarded-fabric-tpm-trusted-attestation-capturing-hardware.md [hunk 2]
```diff
@@ -135,7 +135,8 @@ For more information about the available CI policy rule levels, see [Deploy code
     ```powershell
     Add-HgsAttestationCIPolicy -Path <Path> -Name '<PolicyName>'
     ```
-
+    > [!NOTE]
+    > If you're using a signed code integrity policy, register an unsigned copy of the same policy with HGS. The signature on code integrity policies is used to control updates to the policy, but is not measured into the host TPM and therefore cannot be attested to by HGS.
```
Comment (illfated) [inline WindowsServerDocs/security/guarded-fabric-shielded-vm/guarded-fabric-tpm-trusted-attestation-capturing-hardware.md:139]: Cosmetic/codestyle: please keep a blank line margin before and after any Note blob in the document text, just to be on the safe side:
```suggestion

    > [!NOTE]
    > If you're using a signed code integrity policy, register an unsigned copy of the same policy with HGS. The signature on code integrity policies is used to control updates to the policy, but is not measured into the host TPM and therefore cannot be attested to by HGS.

```

**Comment by carlosmayol** [comment]
Closing it

**Comment by carlosmayol** [comment]
@illfated hello I just approved the suggested modification, please can you push the change? thanks

**Comment by illfated** [comment]
Thank you for having faith in me being employed with MS or part of the MS Docs team here on GitHub. Also, I am sorry for making it look that way. I am not a part of the team. I am mostly just a contributor, quite familiar with how things work around here.

First of all, you need to reopen this PR to make it mergeable (yes, I know that "close and comment" button seems too easy to select instead of the regular "Comment" button. When this PR is open, it becomes mergeable to the MS Docs Server team members. Otherwise, it may look like something you threw in the waste bin and did not want to be merged.

**Comment by carlosmayol** [comment]
> Thank you for having faith in me being employed with MS or part of the MS Docs team here on GitHub. Also, I am sorry for making it look that way. I am not a part of the team. I am mostly just a contributor, quite familiar with how things work around here.
> 
> First of all, you need to reopen this PR to make it mergeable (yes, I know that "close and comment" button seems too easy to select instead of the regular "Comment" button). When this PR is open, it becomes mergeable to the MS Docs Server team members. Otherwise, it may look like something you threw in the waste bin and did not want to be merged.

Yeah I realized after, sorry we work with some external help and I thought you were part of the team, for some reason the chnage was not pushed out but I deleted my local fork. Created a new PR  already 👍

**Comment by illfated** [comment]
Sounds good to me. Thank you for the update.

**Comment by illfated** [comment]
( ref. #5432 )

**Pull request closed without merging**